<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="ButtonBoxWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>571</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>184</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="scanFileMenuButton">
        <property name="text">
         <string>Сканирование файла</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="scanFolderMenuButton">
        <property name="text">
         <string>Сканирование папки</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="directoryMonitoringMenuButton">
        <property name="text">
         <string>Мониторинг директории</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="scheduleScanMenuButton">
        <property name="text">
         <string>Сканирование по расписанию</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteMaliciousFilesMenuButton">
        <property name="text">
         <string>Удаление вредоносных файлов</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quarantineMenuButton">
        <property name="text">
         <string>Карантин</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>141</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../icons.qrc">:/images/resources/antivirus.ico</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="mainWidget" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>561</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QWidget" name="welcomeWidget" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>491</width>
       <height>151</height>
      </rect>
     </property>
     <widget class="QLabel" name="welcomeLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>381</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Добро пожаловать в Касперский v2.0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="scanFileWidget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>491</width>
       <height>221</height>
      </rect>
     </property>
     <widget class="QPushButton" name="scanFileButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Сканировать</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="scanFileEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>109</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="scanFileLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Введите название файла для сканирования:</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="meowLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>361</width>
       <height>491</height>
      </rect>
     </property>
     <property name="pixmap">
      <pixmap resource="../icons.qrc">:/movie/resources/meow.gif</pixmap>
     </property>
    </widget>
    <widget class="QWidget" name="scanFolderWidget" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>521</width>
       <height>501</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
